terraform/main.tf

```diff
@@ -245,4 +244,19 @@ module "webapp-logs-s3" {
   application_name    = "webapp"
 }
 
+module "acm" {
+  source      = "./modules/acm"
+  ssl_domains = var.ssl_domains
+}
+
+module "route53" {
+  source                    = "./modules/route53"
+  root_domain_name          = var.root_domain_name
+  a_records                 = var.a_records
+  webapp_alb_dns            = module.webapp-alb.alb-dns
+  backoffice_alb_dns        = module.backoffice-alb.alb-dns
+  domain_validation_options = module.acm.domain_validation_options
+  ssl_certificate_arn       = module.acm.ssl_certificate_arn
 
+  depends_on = [module.acm, module.webapp-alb, module.backoffice-alb]
```

Comment:
I think terraform is clever enough to know that this module depends on the others. It's rare to need a depends on

Reply:
Thanks. I've removed this and some others without changing the plan output.